src/dropin.js

```diff
@@ -219,7 +219,7 @@ HAS_RAW_PAYMENT_DATA[constants.paymentMethodTypes.applePay] = true;
  */
 
 /**
- * This event is emitted when the payment method available in Drop-in changes. This includes when the state of Drop-in transitions from having no payment method available to having a payment method available and when the payment method available changes. This event is not fired if there is no payment method available on initialization. To check if there is a payment method requestable on initialization, use {@link Dropin#isPaymentMethodRequestable|`isPaymentMethodRequestable`}.
+ * This event is emitted when the payment method available in Drop-in changes. This includes when the state of Drop-in transitions from having no payment method available to having a payment method available and when the payment method available changes. This event is not fired if there is no payment method available on initialization. Note: auto-submit behavior should not be dependent on this event. To check if there is a payment method requestable on initialization, use {@link Dropin#isPaymentMethodRequestable|`isPaymentMethodRequestable`}.
```

Comment:
Auto submit behavior is the exact case for use here, especially for non-cc payment methods. CC payment methods can do it too, just need to be careful if the fraud rules are set up to include postal code, since it'll fire too early.

The code example specifically shows how to check if it's a payment method that should be allowed to submit automatically.